blog/content/post/2023-01-23-3mdeb-insights-winter-2023.md

```diff
@@ -0,0 +1,122 @@
+---
+title: 3mdeb Insights - Winter 2023
+abstract: 'This blog will provide an in-depth look at upcoming plans and events
+          for the near future.'
+cover: /covers/image-file.png
+author: norbert.kaminski
+layout: post
+published: true
+date: 2023-01-23
+archives: "2023"
+
+tags:
+  - Insights
+  - Events
+  - Open-source contribution
+categories:
+  - Firmware
+  - IoT
+  - Miscellaneous
+  - OS Dev
+  - App Dev
+  - Security
+  - Manufacturing
+
+---
+
+
+This blog will provide an in-depth look at upcoming plans and events for the
+near future. In the next quarter, we have several exciting events planned
+for the 3mdeb community. Our events aim to provide a platform for members to
+connect, share knowledge, and stay informed about the latest developments in
+the open-source ecosystem.
+
+## Upcoming events
+
+![Fosdem 2023 logo](/img/FOSDEM23.jpeg)
+
+### FOSDEM 2023
+
+FOSDEM is a large, annual event for open-source software developers that
+takes place in Brussels, Belgium on February 4th and 5th. It is an opportunity
+for developers from all over the world to come together, share ideas, and
+learn about the latest open-source technologies. 3mdeb will be participating
+in FOSDEM and will be hosting an "Open Source Firmware, BMC, and Bootloader
+Devroom" to showcase their contributions to the open-source ecosystem and
+connect with other developers. Additionally, 3mdeb engineers will be present
+to share their knowledge with attendees.
```

Comment:
It is not fair to take all the credits for organization. We should mention ourselves last not first. We organize it with Daniel Kiper and question for him is if we can officially mention we organize devroom with Oracel, or rather Oracle organize this devroom with us.